<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenrenStatusWidget</class>
 <widget class="QWidget" name="RenrenStatusWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>USense</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resource.qrc">
    <normaloff>:/images/USenseIcon.png</normaloff>:/images/USenseIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QToolButton" name="toolButton_Emotion">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/images/sina/emotion.png</normaloff>:/images/sina/emotion.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_Wordscount">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_Mention">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/images/sina/at.png</normaloff>:/images/sina/at.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_Cancel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>71</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/images/sina/cancel.png</normaloff>:/images/sina/cancel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_Send">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>50</y>
     <width>71</width>
     <height>53</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../../Resource.qrc">
     <normaloff>:/images/sina/ok.png</normaloff>:/images/sina/ok.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Emotions">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>230</y>
     <width>611</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>表情</string>
   </property>
   <widget class="QListWidget" name="listWidget_Emotions">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>251</width>
      <height>170</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>默认</string>
     </property>
    </item>
   </widget>
   <widget class="QWebView" name="webView_Emotions">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>281</width>
      <height>170</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit_Statuscontent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>59</y>
     <width>520</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../Resource.qrc">:/images/renren/新鲜事.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#title_label
{
	color:rgb(255, 255, 255);
	font-size:20px;
	font-family:&quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
